Added description use rclone (#74)
near/node-docs: docs/intro/snapshots.md

For a faster snapshot download speed, use s5cmd, the download accelerator for S3 written in Go. For download instruction, please see https://github.com/peak/s5cmd.

> Next alternative you can use `rclone`. 

Reviewer: ```suggestion
Another alternative is [`rclone`](https://rclone.org). 
```

This tool is present in many Linux distributions. There is also a version for Windows.
And its main merit is multithread.
You can [read about it on](https://rclone.org)

First, install rclone:
```
$ sudo apt install rclone
```

> Next, prepare config. Because interactivity config very long and you must know meny parameters.

Reviewer: ```suggestion
Next, prepare config, so you don't need to specify all the parameters interactively:
```

```
mkdir -p ~/.config/rclone
touch ~/.config/rclone/rclone.conf
```

> ... and paste next rows into file rclone.conf 

Reviewer: ```suggestion

, and paste the following config into `rclone.conf`:
```

```
[near_s3]
type = s3
provider = AWS
location_constraint = EU
acl = public-read
server_side_encryption = AES256
region = ca-central-1
```
Next step very similar with aws-cli. 

> >With two exceptions.
> 1. By default rclone use 4 threads. You can change this by adding key `--transfers=X` where X is count of threads.
> 2. You can view or not state of downloading. This key is `--progress`

Reviewer: I feel this is not relevant, and whoever would need to dig into it will learn more about `rclone` separately, so to keep the page short, I will remove this part.

```suggestion
```
